faunadb-java/src/main/java/com/faunadb/client/FaunaClient.java

```diff
@@ -330,10 +351,22 @@ private Value handleResponse(HttpResponse<String> response) {
     }
   }
 
+  private MetricsResponse handleResponseWithMetrics(HttpResponse<String> response) {
+    Map<MetricsResponse.Metrics, Optional<String>> metrics = MetricsResponse.Metrics.stream()
```

Comment:
Do you want to keep the `Optional` here?

Reply:
yep, I've removed this for consistency with scala as well